sockjs/handler_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	"net/url"
 	"testing"
 	"time"
+
+	"github.com/stretchr/testify/require"
```

Comment:
This is a new dependency right? I am not saying this is bad as `testify/require` is common these days, just noticed that it was not used before.

Reply:
I removed testify and added `FailNow()`. Not worth adding new dependency just for couple of uses.